src/applications/vaos/project-cheetah/index.jsx

```diff
@@ -20,13 +22,28 @@ import ConfirmationPage from './components/ConfirmationPage';
 import { selectFeatureProjectCheetah } from '../redux/selectors';
 import ReceivedDoseScreenerPage from './components/ReceivedDoseScreenerPage';
 import ContactFacilitiesPage from './components/ContactFacilitiesPage';
+import { FETCH_STATUS } from '../utils/constants';
+import LoadingIndicator from '@department-of-veterans-affairs/component-library/LoadingIndicator';
 import useManualScrollRestoration from '../hooks/useManualScrollRestoration';
 import useFormRedirectToStart from '../hooks/useFormRedirectToStart';
 import useFormUnsavedDataWarning from '../hooks/useFormUnsavedDataWarning';
 
-export function NewBookingSection({ featureProjectCheetah }) {
+export function NewBookingSection({
+  windowsStatus,
+  featureProjectCheetah,
+  openNewBookingPage,
+  pageChangeInProgress,
+}) {
   const match = useRouteMatch();
   const history = useHistory();
+  const location = useLocation();
+
+  useEffect(
+    () => {
+      openNewBookingPage(history);
+    },
+    [pageChangeInProgress],
```

Comment:
Follow up to in person walkthrough: this results in making the api call to get the settings every time you changes pages in the flow. There are a few different ways of solving this, one way might be to add the `isEligible` property in your `mapDispatchToProps`. That way, after all the `use` function calls, you could do something like:

```
if (!isEligible && windowsStatus === FETCH_STATUS.succeeded && !location.pathname.includes('contact-facilities')) {
  return <Redirect to="/new-project-cheetah-booking/contact-facilities" />;
}
```

Ideally, I think we want `openNewBookingPage` called just once, and then to use the `isEligible` state after that.